helm/charts/cytomine/templates/pims/pims.yaml

```diff
@@ -28,7 +28,7 @@ spec:
       securityContext:
         runAsUser: {{ .Values.containerSecurity.userID }}
         fsGroup: {{ .Values.containerSecurity.fsgroupID }}
-      serviceAccountName: {{ include "cytomine.serviceAccountName" . }}
+      # serviceAccountName: {{ include "cytomine.serviceAccountName" . }}
```

Comment:
You remove it in core.yaml but commented here, is this needed? Otherwise, I suggest to remove it.